src/parser/utils.ts

```diff
@@ -27,3 +27,24 @@ export function isMap(d: unknown): d is Map<unknown, unknown> {
 export function isDate(d: unknown): d is Date {
   return Object.prototype.toString.call(d) === '[object Date]';
 }
+
+/** @internal */
+export type StylizeFunction = (x: unknown, style: string) => string;
+export type InspectParameterFn = (x: unknown, options: unknown) => string;
+export function getBasicInspectParameterFn(): InspectParameterFn {
+  // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+  return v => `${v}`;
+}
+export function getStylizeFunction(options?: unknown): StylizeFunction {
+  const stylizeExists =
+    options != null &&
+    typeof options === 'object' &&
+    'stylize' in options &&
+    typeof options.stylize === 'function';
+
+  if (stylizeExists) {
+    return options.stylize as (x: unknown, style: string) => string;
```

Comment:
again, super tiny nit, feel free to ignore, but since you already have the type defined:

```suggestion
    return options.stylize as StylizeFunction;
```

Reply:
Sounds good!